packages/amplify-graphql-api-construct/src/amplify-graphql-api.ts

```diff
@@ -87,6 +87,9 @@ export class AmplifyGraphqlApi extends Construct {
    */
   public readonly resources: AmplifyGraphqlApiResources;
 
+  /**
+   * Reference to root stack of data construct
```

Comment:
```suggestion
   * Reference to parent stack of data construct
```

or `nearest parent` .  "Root stack" might be something different depending on shape of construct tree.